### PR TITLE
blog: hackathon 6 winner article

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

File: apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+![Swag kit](/images/blog/lw6-hackathon/swagkit.png)
+
+The swag kits are limited edition and extremely valuable, so keep them safe! Again, congratulations on all the winners!
+
+## Hang out with us and the winners of the Hackathon
+
+To celebrate the massive success of the Hackathon, we are hosting a Discord Stage Channel with and inviting the winners and everyone who participated. Join us on Thursday, Jan 5th at 7:00 AM PST.
+
+**[Join our Discord](https://discord.supabase.com/)** and **[Set a reminder](https://twitter.com/i/spaces/1LyxBoXywLjKN)** for the event.
```
Comment (dshukertjr) [inline apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx:120]: @silentworks Could you update the link for `Set a reminder` once the Discord Stage channel is ready?

File: apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Each members of the winners in other categories will receive a limited edition supaverified swag kit, and the winners of the Best Edge Functions projects will receive swag kits from the Deno team.
+
+![Swag kit](/images/blog/lw6-hackathon/swagkit.png)
+
+The swag kits are limited edition and extremely valuable, so keep them safe! Again, congratulations on all the winners!
+
+## Hang out with us and the winners of the Hackathon
+
+To celebrate the massive success of the Hackathon, we are hosting a Discord Stage Channel with and inviting the winners and everyone who participated. Join us on Thursday, Jan 5th at 7:00 AM PST.
```
Comment (dshukertjr) [inline apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx:118]:
```suggestion
To celebrate the massive success of the Hackathon, we are hosting a Discord Stage Channel and inviting the winners and everyone who participated. Join us on Thursday, Jan 5th at 7:00 AM PST.
```

File: apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+title: 'Launch Week 6 Hackathon Winners'
+description: Announcing the winners of the Launch Week 6 Hackathon!
+author: tyler_shukert
```
Comment (dshukertjr) [inline apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx:4]:
```suggestion
author: andrew_smith,tyler_shukert
```

File: apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+### Winner
+
+[supaquiz - Multiplayer trivia/quiz game](https://github.com/yallurium/supaquiz) - by [@yallurium](https://twitter.com/yallurium).
+
+Supaquiz is a realtime multiplayer trivia/quiz game where you can host or join gmaes with other players remotely.
+
+![supaquiz - Multiplayer trivia/quiz game](/images/blog/lw6-hackathon-winners/supaquiz.png)
+
+### Runner Up
+
```
Comment (dshukertjr) [inline apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx:67]:
```suggestion
```

**Review by RamiroND — APPROVED** [review]

**Pull request merged**